transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/utility/PathUtils.java

```diff
@@ -0,0 +1,9 @@
+package com.qpay.transactionhistorymanager.utility;
+
+public final class PathUtils {
+    public static final String HISTORY_PATH = "/api/v1/transactions";
+
+    private PathUtils() {
+        throw new java.lang.UnsupportedOperationException("This is a utility class and cannot be instantiated");
+    }
```

Comment:
https://medium.com/@sharadblog/private-constructor-in-java-with-lombok-ff5f4765b62d